packages/vscode-extension/src/node/shopify-magic-prompts/html.ts

```diff
@@ -0,0 +1,14 @@
+export function htmlRules() {
+  return `
+<html>
+  - Use semantic HTML
+  - Use modern HTML features where appropriate, e.g. use \`<details>\` and \`<summary>\` over JS to show and hide content
+  - Use \`CamelCase\` for IDs. When appending a block or section ID, append \`-{{ block.id }}\` or \`-{{ section.id }}\` respectively
```

Comment:
Did you mean this?
```suggestion
  - Use \`camelCase\` for IDs. When appending a block or section ID, append \`-{{ block.id }}\` or \`-{{ section.id }}\` respectively
```

Reply:
Actually, I believe this is [accurate](https://github.com/Shopify/dawn/blob/bbfd08b39f12a2193606a973eebcb2d15bc8d957/layout/theme.liquid#L312) 😌

Reply:
Oh, in that case I think we should call it [PascalCase](https://www.freecodecamp.org/news/snake-case-vs-camel-case-vs-pascal-case-vs-kebab-case-whats-the-difference/#heading-what-is-pascal-case)? Or are these things called differently in JS and Ruby worlds?

<img width="214" alt="image" src="https://github.com/user-attachments/assets/a9b1b0dc-d6b3-4d06-b6ed-e3f98edb6146" />

```suggestion
  - Use \`PascalCase\` for IDs. When appending a block or section ID, append \`-{{ block.id }}\` or \`-{{ section.id }}\` respectively

```